main.nf

```diff
@@ -9,38 +9,56 @@
 ----------------------------------------------------------------------------------------
 */
 
+nfcore_logo = """=======================================================
+                                          ,--./,-.
+          ___     __   __   __   ___     /,-._.--~\'
+    |\\ | |__  __ /  ` /  \\ |__) |__         }  {
+    | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
+                                          `._,._,\'
+
+nf-core/rnafusion v${workflow.manifest.version}"
+======================================================="""
+
 def helpMessage() {
-    log.info"""
-    =========================================
-     nf-core/rnafusion v${workflow.manifest.version}
-    =========================================
+    log.info """${nfcore_logo}
```

Comment:
I have a nasty feeling that this will break the whitespace when running `--help`. The logo doesn't have an indentation at the start of those lines, which means that the `.stripIndent()` call won't properly remove the indentation from the rest of the block.

Reply:
It works, I've tested it before 😉.
```
nextflow run . --help                                                                                                                                     [16:09:42]
N E X T F L O W  ~  version 18.10.1
Launching `./main.nf` [romantic_meitner] - revision: 2ba29c5345
=======================================================
                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~'
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

nf-core/rnafusion v1.0"
=======================================================
    Usage:

    The typical command for running the pipeline is as follows:

    nextflow run nf-core/rnafusion --reads '*_R{1,2}.fastq.gz' -profile docker

    Mandatory arguments:
      --reads                       Path to input data (must be surrounded with quotes)
      -profile                      Configuration profile to use. Can use multiple (comma separated)
                                    Available: standard, conda, docker, singularity, awsbatch, test

    Tool flags:
      --star_fusion                 Run STAR-Fusion
      --fusioncatcher               Run FusionCatcher
        --fc_extra_options          Extra parameters for FusionCatcher. Can be found at https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md
      --fusion_inspector            Run Fusion-Inspector
      --ericscript                  Run Ericscript
      --pizzly                      Run Pizzly
      --squid                       Run Squid
      --test                        Runs only specific fusion tool/s and not the whole pipeline. Only works on tool flags.
      --tool_cutoff                 Number of tools to pass threshold required for showing fusion in the report. [Default = 2]

    References                      If not specified in the configuration file or you wish to overwrite any of the references.
      --fasta                       Path to Fasta reference
      --gtf                         Path to GTF annotation
      --star_index                  Path to STAR-Index reference
      --star_fusion_ref             Path to STAR-Fusion reference
      --fusioncatcher_ref           Path to Fusioncatcher reference
      --fusion_inspector_ref        Path to FusionInspector reference
      --ericscript_ref              Path to Ericscript reference
      --pizzly_fasta                Path to Pizzly FASTA reference
      --pizzly_gtf                  Path to Pizzly GTF annotation

    Options:
      --genome                      Name of iGenomes reference
      --singleEnd                   Specifies that the input is single end reads

    Other Options:
      --outdir                      The output directory where the results will be saved
      --email                       Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits
      -name                         Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.

    AWSBatch options:
      --awsqueue                    The AWSBatch JobQueue that needs to be set when running on AWSBatch
      --awsregion                   The AWS Region for your AWS Batch job to run on

```

Reply:
Ahh, the usage should not have indentation?

Reply:
Correct 👍

Reply:
If you set the help text into a variable and run `stripindent()` first, then log the concatenated variables you should be ok..

Reply:
What if I just put 
```
log.info ${nfcore_logo}
log.info ""Usage ...".stripIndent()
```

Reply:
Also fine 👍

Reply:
Solved.